app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -624,6 +657,26 @@ class BrowserTabViewModel(
         registerSiteVisit()
     }
 
+    private fun sendPixelIfUserOurAppSiteVisitedFirstTime(url: String) {
```

Comment:
small typo. It should be `...IfUseOurApp...`not User

Reply:
If I got a penny every time I've typed user instead of use I would have at least 3 pennies. Done!